tests/scripts/all.sh

```diff
@@ -3594,6 +3594,88 @@ component_test_psa_crypto_config_accel_aead () {
     make test
 }
 
+# This is a common configuration function used in:
+# - component_test_psa_crypto_config_accel_cipher_aead
+# - component_test_psa_crypto_config_reference_cipher_aead
+common_psa_crypto_config_accel_cipher_aead() {
+    scripts/config.py -f "$CRYPTO_CONFIG_H" unset PSA_WANT_ALG_CMAC
+    scripts/config.py -f "$CRYPTO_CONFIG_H" unset PSA_WANT_ALG_STREAM_CIPHER
```

Comment:
Similarly, after #8315 I don't think we should be disabling STREAM or ECB, but rather declare them as accelerated.